src/expressions/plyql.pegjs

```diff
@@ -940,7 +968,7 @@ InSetLiteralExpression
 StringOrNumber = String / Number
 
 String "String"
-  = "'" chars:NotSQuote "'" _ { return chars; }
+  = CharsetIntroducer? "'" chars:NotSQuote "'" _ { return chars; }
   / "'" chars:NotSQuote { error("Unmatched single quote"); }
```

Comment:
Lol I <3 CharsetIntroducer. can you add it to the next line also so we can parse `n'lol` as an error